include/flamegpu/model/AgentDescription.h

```diff
@@ -292,6 +292,10 @@ void AgentDescription::newVariable(const std::string &variable_name, const std::
         THROW exception::ReservedName("Agent variables cannot be named 'name' or 'state', these are reserved for backwards compatibility reasons, "
             "in AgentDescription::newVariable().");
     }
+    if (lower_variable_name == "fgpu2_reserved_bin_index") {
```

Comment:
Reserved variables should begin with `_`

Reply:
@ptheywood also wanted to move to using `constexpr` for storing string literals?

Reply:
`constexpr` is better than `const` for string literals, though most of the benefits aren't available until C++17 (or 20+ really, but that's not happening any time soon). 

Also strip out `fgpu2` from the var name please